plugins/formal-verification/README.md

```diff
@@ -33,10 +33,16 @@ A few notes:
 
 ## Directory structure:
 
-- `src/org/.../formver/plugin`: Infrastructure for integrating our code into the compiler
-  and reporting warnings.
-- `src/org/.../formver/conversion`: Conversion logic from Kotlin to Viper.
-- `src/org/.../formver/scala`: Utilities for constructing Viper data structures, which are defined in Scala.
+- `formver.embeddable`: Separate target necessary for the Gradle plugin.
+- `formver.cli`: Command-line processor and plugin loading implementation.
+- `formver.core`: Core conversion logic and associated data structures.
+  - `formver.core/.../conversion`: Conversion logic from the FIR to Viper
+  - `formver.core/.../embeddings`: Embedded versions of Kotlin constructs.
+  - `formver.core/.../domains`: Built-in domains used in the conversion.
```

Comment:
The domains are now only in `formver.viper` and not in `formver.core` anymore right?

Reply:
Right, sorry, copy-paste error.  Fixed!